src/domain/common/haittaIndexes/HaittaIndexNumber.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box, VisuallyHidden } from '@chakra-ui/react';
+import {
+  LIIKENNEHAITTA_STATUS,
+  getStatusByIndex,
+  getColorByStatus,
+  TormaysIndex,
+} from '../utils/liikennehaittaindeksi';
+
+type Props = { index?: TormaysIndex; testId?: string };
+
+const HaittaIndexNumber: React.FC<Props> = ({ index, testId }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Box
+      backgroundColor={getColorByStatus(getStatusByIndex(index))}
+      color={getStatusByIndex(index) === LIIKENNEHAITTA_STATUS.YELLOW ? 'black' : 'white'}
```

Comment:
Should `getStatusByIndex(index)` be called only once and used the value in these two places?

Reply:
Good point. Fixed that.